fiat/prices.go

```diff
@@ -91,17 +93,56 @@ var priceBackendNames = map[PriceBackend]string{
 	CoinDeskPriceBackend: "coindesk",
 }
 
+var priceBackendProxySupport = map[PriceBackend]bool{
```

Comment:
thoughts on doing a config map instead of the 2 separate `priceBackendNames` and `priceBackendProxySupport` maps? So something like this:

```
type backendsConfig struct {
	name string
	proxySupport bool
}

var priceBackendsConfig = map[PriceBackend]backendsConfig {
	UnknownPriceBackend: {
		name: "unknown",
		proxySupport: false,
        },
	CoinCapPriceBackend: {
		name: "coincap",
		proxySupport: false,
	},
	CoinDeskPriceBackend: {
		name: "coindesk",
		proxySupport: true,
	},
}
```